renku/core/interface/storage.py

```diff
@@ -90,6 +95,5 @@ def mount(self, path: Union[Path, str]) -> None:
         raise NotImplementedError
 
     @abc.abstractmethod
-    def set_configurations(self) -> None:
-        """Set required configurations to access the storage."""
-        raise NotImplementedError
+    def run_rclone_command(self, command: str, uri: str, *args, **kwargs) -> Any:
+        """Run a RClone command by possibly add storage-specific flags."""
```

Comment:
No `raise NotImplementedError` here?

Reply:
I've refactored this in https://github.com/SwissDataScienceCenter/renku-python/pull/3163 and it's removed from the interface. So, this will be addressed once that PR is merged if it's ok with you.